test/e2e/data/infrastructure-docker/v1beta1/main/bases/cluster-with-topology.yaml

```
@@ -26,6 +26,7 @@ spec:
         name: "md-0"
         nodeDeletionTimeout: "30s"
         nodeVolumeDetachTimeout: "5m"
+        minReadySeconds: 30
```

```suggestion
        minReadySeconds: 5
```
nit. Not sure if that property infuences test speed in some way. Thus I would prefer a relatively low value